tests/Collections/PrefixTrieTests.cs

```diff
@@ -112,16 +109,13 @@ public void TestAllKeys()
 
             var withPrefixO = trie.AllKeys("o").ToList();
 
-            Assert.Equal(1, withPrefixO.Count);
-            Assert.Equal("one", withPrefixO[0]);
+            withPrefixO.Should().HaveCount(1);
+            withPrefixO.Should().Contain("one");
 
             var withPrefixTh = trie.AllKeys("th").ToList();
 
-            Assert.Equal(2, withPrefixTh.Count);
-            foreach (var word in words.Where(w => w.StartsWith("th")))
-            {
-                Assert.True(withPrefixTh.Contains(word));
-            }
+            withPrefixTh.Should().HaveCount(2);
+            withPrefixTh.Should().Contain(words.Where(w => w.StartsWith("th")));
```

Comment:
```suggestion
            withPrefixTh.Should().BeEquivalentTo(words.Where(w => w.StartsWith("th")));
```

Don't need the `HaveCount` then. Same with the test below.